src/lematerial_forgebench/benchmarks/distribution_benchmark.py

```diff
@@ -0,0 +1,155 @@
+    ):
+        """Initialize the distribution benchmark.
+
+        Parameters
+        ----------
+        name : str
+            Name of the benchmark.
+        distribution_functions : list[str], optional
+            A list of strings containing the distribution functions to compare. If none, 
+            defaults to all currently encoded. 
+        description : str, optional
+            Description of the benchmark.
+        metadata : dict, optional
+            Additional metadata for the benchmark.
+        """
+        if description is None:
+            description = (
+                "Evaluates the stability and metastability of crystal structures"
+            )
+
+        # Initialize the JSDistance metric
+        JSDistance_metric = JSDistance()
+
+        # Set up evaluator configs
+        evaluator_configs = {
+            "JSDistance": EvaluatorConfig(
+                name="JSDistance",
+                description="Calculates the JS Distance between two distributions",
+                metrics={"JSDistance": JSDistance_metric},
+                weights={"JSDistance": 1.0},
+                aggregation_method="weighted_mean",
+            ),
+        }
+
+        # Initialize the MMD metric
+        MMD_metric = MMD()
+        evaluator_configs["MMD"] = EvaluatorConfig(
+            name="MMD Analysis",
+            description="Calculates the MMD between two distributions",
+            metrics={"MMD": MMD_metric},
+            weights={"MMD": 1.0},
+            aggregation_method="weighted_mean",
+        )
+
+        # FrechetDistance_metric = FrechetDistance()
+        # evaluator_configs["FrechetDistance"] = EvaluatorConfig(
+        #     name="FrechetDistance Analysis",
+        #     description="Calculates the Frechet Distance between two distributions",
+        #     metrics={"FrechetDistance": FrechetDistance_metric},
+        #     weights={"FrechetDistance": 1.0},
+        #     aggregation_method="weighted_mean",
+        # )
+
+
+        # Create benchmark metadata
+        benchmark_metadata = {
+            "version": "0.1.0",
+            "category": "distribution",
+            **(metadata or {}),
+        }
+
+        super().__init__(
+            name=name,
+            description=description,
+            evaluator_configs=evaluator_configs,
+            metadata=benchmark_metadata,
+        )
+
+    def aggregate_evaluator_results(
+        self, evaluator_results: Dict[str, EvaluationResult]
+    ) -> Dict[str, float]:
+        """Aggregate results from multiple evaluators into final scores.
+
+        Parameters
+        ----------
+        evaluator_results : dict[str, EvaluationResult]
+            Results from each evaluator.
+
+        Returns
+        -------
+        dict[str, float]
+            Final aggregated scores.
+        """
+        import math
+
+        def safe_float(value):
+            """Safely convert value to float, handling None and NaN."""
```

Comment:
What's the intention here with `safe_float`? What is supposed to happen to `None` and `nan`?